apps/web/components/AddressForm/AddressForm.vue

```diff
@@ -101,10 +101,10 @@ import type { AddressFormProps } from './types';
 const { loading: loadBilling } = useBillingAddress();
 const { loading: loadShipping } = useShippingAddress();
 const props = defineProps<AddressFormProps>();
-const isCartUpdateLoading = computed(() => loadBilling.value || loadShipping.value);
+const isCartUpdateLoading = computed(() => props.loading || loadBilling.value || loadShipping.value);
 
 const { savedAddress } = toRefs(props);
-const useAsShippingAddress = ref(true);
+const useAsShippingAddress = ref(false);
```

Comment:
Why "false" here?
We said we want that box checked ([PRODUCT BACKLOG ITEM 72842](https://dev.azure.com/plentymarkets/plentymarkets/_workitems/edit/72842))

Reply:
Than we have to implement that as a prop, that we can handle both cases.

Reply:
From what I recall, part of the reason why we had that specification was simplicity. It's fine if the box is checked every time the modal opens I'd say.

Reply:
I talked with Tim, it should be different behaviors in the myaccount and checkout, I added a prop with default to true.